proto/one/operations/spreadsheet/computation_binding.proto

```diff
@@ -29,6 +29,9 @@ message ComputationBinding {
    // details on when to execute computation and where to put the output defaults to every row
    ExecutionDetails executionDetails = 7;
 
+   // expression text of the computation
+   google.protobuf.StringValue expression = 8;
```

Comment:
I think you will need to import the google.protobuf.wrappers library in order for this to build. Check another model that uses this type for an example.